adding_species_manuscript.tex

```diff
@@ -226,14 +222,14 @@ \section*{Introduction}
 more and more detailed elements of a species' biology.
 Important elements include genomic features such as mutation and recombination
 rates that strongly affect genetic variation and haplotype structure
-\citep{Nachman2002}. These have particularly strong ramifications 
+\citep{Nachman2002}, which have particularly strong ramifications 
 when linked selection is important in the patterns of genomic diversity being studied \citep{Cutter2013}.
 Furthermore, the demographic history of a species,
 encompassing population sizes and distributions, divergences, and gene flow, can
 dramatically affect patterns of genomic variation \citep{Teshima2006}. Thus
 species-specific estimates of these and other ecological and evolutionary parameters 
 (e.g., those governing the process of natural selection) 
-are fundamentally important when developing simulations.
+are important to realistic simulations.
```

Comment:
I don't think they're quite "fundamental" (i.e., everything else is built on them).